action.yaml

```diff
@@ -4,13 +4,17 @@ inputs:
   pattern:
     description: 'Patterns to checkout'
     required: true
+  lfs:
+    description: 'Whether to checkout LFS files (true/false)'
+    default: 'false'
 
 runs:
   using: 'composite'
   steps:
     - run: |
         REPO="https://${GITHUB_ACTOR}:${{ github.token }}@github.com/${GITHUB_REPOSITORY}.git"
         BRANCH="${GITHUB_REF/#refs\/heads\//}"
+        GIT_LFS_SKIP_SMUDGE=${{ ! inputs.lfs }}
```

Comment:
Strings are coerced to numbers
https://docs.github.com/en/actions/learn-github-actions/expressions#operators

Reply:
Well, apparently not